container/utils.py

```diff
@@ -57,23 +58,47 @@ def jinja_render_to_temp(template_file, temp_dir, dest_file, **context):
     open(os.path.join(temp_dir, dest_file), 'w').write(
         rendered.encode('utf8'))
 
+
 def get_config(base_path, var_file=None):
     return AnsibleContainerConfig(base_path, var_file=var_file)
 
+
 def config_format_version(base_path, config_data=None):
     if not config_data:
         config_data = get_config(base_path)
     return int(config_data.pop('version', 1))
 
-def assert_initialized(base_path):
-    ansible_dir = os.path.normpath(
-        os.path.join(base_path, 'ansible'))
+
+def assert_initialized(base_path, kwargs):
```

Comment:
Using `kwargs` here makes for a messy method signature - since it's `playbook` that you're interested in as a kwarg, just use `playbook='main.yml'` here.